_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -923,6 +930,112 @@ public function update_data( $request ) {
 	}
 
 	/**
+	 * Perform tasks in the site based on onboarding choices.
+	 *
+	 * @since 5.3
```

Comment:
@eliorivero can you make sure to update all of these to 5.4.0?

Reply:
Thanks @dereksmart. Done in bc1101b